appveyor.yml

```diff
@@ -76,3 +81,9 @@ deploy: off
 on_finish:
 # Set blockRdp to true to allow RDP, uses Git tag build-agent-v6.1.0%2B1300
   - ps: $blockRdp = $false; iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/ab48d6e110f2feb585b6b908a72d6e7dbfb83cb3/scripts/enable-rdp.ps1'))
+
+artifacts:
+  - name: Windows Release
+    path: aeternity-windows-w64-*.exe
```

Comment:
Could you please change the naming to match our current scheme?
It's `aeternity-$version-$osname-$arch-*` Where arch is `x86_64` for 64bit architectures.

Reply:
Of course leave it as is if it's too much hassle

Reply:
Changed it.